packages/core/contracts/common/implementation/Testable.sol

```diff
@@ -7,7 +7,7 @@ import "./Timer.sol";
  * @title Base class that provides time overrides, but only if being run in test mode.
  */
 abstract contract Testable {
-    // If the contract is being run on the test network, then `timerAddress` will be the 0x0 address.
```

Comment:
OZ comment:
```
In the Testable contract there is an inline comment that says "If the contract is being run on the test network, then timerAddress will be the 0x0 address." However, it appears that this comment is erroneous. The onlyIfTest modifier requires that timerAddress != address(0x0). There are also other comments in the codebase that agree with the modifier: "_timerAddress ... Set to 0x0 in production.". The comment in Testable should be modified to agree with the implementation to avoid potential confusion around such a critically important value.
```